src/components/QrScanner/QrCamera.test.tsx

```diff
@@ -11,7 +11,10 @@ jest.mock("expo-permissions", () => ({
 
 const mockPlatform = (platform: "android" | "ios"): void => {
   jest.resetModules();
-  jest.doMock("Platform", () => ({ OS: platform, select: jest.fn() }));
+  jest.doMock("react-native/Libraries/Utilities/Platform", () => ({
```

Comment:
Is there a resource that mentions the background of this change?

Reply:
https://altany.github.io/react-native/0.61/jest/mocking/upgrade/2020/01/25/mocking-react-native-0.61-modules-with-jest.html

Reply:
This was something I found, RN 0.61 removes haste maps